lib/jnpr/junos/decorators.py

```diff
@@ -73,3 +75,53 @@ def wrapper(*args, **kwargs):
                 raise
 
     return wrapper
+
+
+def ignoreWarnDecorator(function):
+    """
+    Ignore warning if ignore_warning provided and the rpc-reply severity level
+    is warning
+
+    For example::
+        dev.rpc.get(ignore_warning=True)
+        dev.rpc.get(ignore_warning='vrrp subsystem not running')
+        dev.rpc.get(ignore_warning='vrrp subsystem not running', 'statement not found')
+        cu.load(cnf, ignore_warning='statement not found')
+
+    :ignore_warning: It can take take boolean value or string or list of string.
+        if True, it will ignore all warning. If sring, it will ignore warning if
+        the statement matches given string. If list of strings, it will try to check
+        if warning statement is from any of the given strings in the list:
+    """
+    @wraps(function)
+    def wrapper(*args, **kwargs):
+        if 'ignore_warning' in kwargs:
+            ignore_warn = kwargs['ignore_warning']
+            try:
+                result = function(*args, **kwargs)
+                return result
+            except RpcError as ex:
+                ex.rpc_xml = JXML.remove_namespaces(ex.rpc_xml)
```

Comment:
In case of multiple rpc errors, if any of the error severity is warning, rpc_xml is set accordingly.
check https://github.com/Juniper/py-junos-eznc/blob/master/lib/jnpr/junos/exception.py#L44-L49